### DISCUSSION TITLE
React Native - Build error on Android

### DISCUSSION THREAD

**Discussion by huyta** [Web3Auth/Web3Auth#808] [Help]
Hi Folks,

I am integrating web3auth RN SDK on android and got bellow error

**Manifest merger failed : Attribute application@name value=(myapp.mobile.MainApplication) from AndroidManifest.xml:19:7-38
	is also present at [org.torusresearch:web3auth-android-sdk:3.0.0] AndroidManifest.xml:17:18-62 value=(com.web3auth.core.Web3AuthApp).
	Suggestion: add 'tools:replace="android:name"' to <application> element at AndroidManifest.xml:7:5-12:19 to override.**

Do you have any suggestion to fix it

Thanks,
Huy

**Comment by yashovardhan** [comment]
Can you share more details on how you got this error? Did you integrate similar to how this example is built?

https://github.com/Web3Auth/examples/tree/main/react-native

**Reply by huyta** [reply]
Hi @yashovardhan,

The example is working well on my side.

I got the above error when trying to integrate web3auth into my exist project (follow by the guideline here https://web3auth.io/docs/sdk/react-native/ )

Thanks,
Huy

**Reply by yashovardhan** [reply]
Can you please mention the react native version you're using? Also according to the error, you need to add a certain element in the android manifest, did you try adding that in `AndroidManifest.xml`?

**Reply by huyta** [reply]
Hi @yashovardhan,
 Fixed it with the suggestion in error message (add 'tools:replace="android:name")

**Reply by yashovardhan** [reply]
haha glad it worked out, I'm marking this as an answer then.
